src/boot_arm.c

```diff
@@ -187,12 +196,23 @@ void isr_reset(void) {
         dst++;
     }
 
-#if !defined(PLATFORM_stm32l5) || (__ARM_FEATURE_CMSE == 0) || (__ARM_FEATURE_CMSE != 3U)  
     mpu_init();
-#endif 
+
     /* Run the program! */
     main();
 }
+
+// forward to app handler
```

Comment:
Perhaps wrap this new ISR forwarder with `#ifdef CORTEX_R5`? Also note the `//` should use `/* */`